.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: General issue
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please fill out the sections below to help us address your issue.
+
+### Version of AWS SDK for PHP?
+
+
+### Version of PHP (`php -v`)?
+
+
+### What issue did you see?
+
+
+### Steps to reproduce
+If you have an runnable example, please include it as a snippet or link to a repository/gist for larger code examples.
+
+### Additional context
+
```

Comment:
Perhaps add a note here for examples of possible context to include, if it may be relevant:
* Framework (Laravel, Wordpress, etc) and version.
* PHP or environment configuration settings, if related to memory/performance